azure-devops/azext_devops/dev/team/team.py

```diff
@@ -0,0 +1,155 @@
+    :param devops_organization: Azure Devops organization URL. Example: https://dev.azure.com/MyOrganizationName/
+    :type devops_organization: str
+    :param project: Name or ID of the project.
+    :type project: str
+    :param detect: When 'On' unsupplied arg values will be detected from the current working
+                   directory's repo.
+    """
+    try:
+        devops_organization, project = resolve_instance_and_project(detect=detect,
+                                                                    devops_organization=devops_organization,
+                                                                    project=project)
+        core_client = get_core_client(devops_organization)
+        return core_client.get_teams(top=top, skip=skip, project_id=project)
+    except VstsServiceError as ex:
+        raise CLIError(ex)
+
+
+def get_team_members(team_id, top=None, skip=None, devops_organization=None, project=None, detect=None):
+    """List members of a particular team.
+    :param team_id: The name or id of the team to show members of.
+    :type team_id: str
+    :param top: Maximum number of members to return.
+    :type top: int
+    :param skip: Number of members to skip.
+    :type skip: int
+    :param devops_organization: Azure Devops organization URL. Example: https://dev.azure.com/MyOrganizationName/
+    :type devops_organization: str
+    :param project: Name or ID of the project.
+    :type project: str
+    :param detect: When 'On' unsupplied arg values will be detected from the current working
+                   directory's repo.
+    """
+    try:
+        devops_organization, project = resolve_instance_and_project(detect=detect,
+                                                                    devops_organization=devops_organization,
+                                                                    project=project)
+        core_client = get_core_client(devops_organization)
+        return core_client.get_team_members(team_id=team_id, top=top, skip=skip, project_id=project)
+    except VstsServiceError as ex:
+        raise CLIError(ex)
+
+
+def update_team(team_id, name=None, description=None, devops_organization=None, project=None, detect=None):
+    """Update a team's name and/or description.
+    :param team_id: The name or id of the team to be updated.
+    :type team_id: str
+    :param name: New name of the team.
+    :type name: str
+    :param description: New description of the team.
+    :type description: str
+    :param devops_organization: Azure Devops organization URL. Example: https://dev.azure.com/MyOrganizationName/
+    :type devops_organization: str
+    :param project: Name or ID of the project.
+    :type project: str
+    :param detect: When 'On' unsupplied arg values will be detected from the current working
+                   directory's repo.
+    :rtype: :class:`<WebApiTeam> <core.v4_0.models.WebApiTeam>`
+    """
+    if name is None and description is None:
```

Comment:
don't see much gain in doing this.
Reasons:
1. This same check is done by API
2. Code in CLI is not operating on these and is simply passing the same to service
3. This should be a corner case (without data here, but this is not the happy path) so for every valid command this check is happening twice (which will slow down every valid command)

I would have recommended to add this check if
we were doing some operation on these variables like splitting, length or anything 
but right now this does not seem useful.

Reply:
i am fine either ways just that it looks like a performance optimization for negative scenario (which might be a good thing) but `if name is None and description is None:` will execute always (for positive scenario) as well which might not be a good thing

Reply:
ok. now I see the point of this check..
it is actually to tell user about requirement rather than fail fast, user can't do mapping from API error to command line

Reply:
Yeah, the exception message from API  won't be enough for the user to know about the missing arguments